portfolio/src/main/webapp/index.html

```diff
@@ -49,8 +49,8 @@ <h2>Gallery</h2>
         </br>
         <h1> Where can you find me? </h1>
             <div id = "links">
-                <a class = "link" href= "https://www.linkedin.com/in/leah-attai-203">LinkedIn</a>
-                <a class = "link" href= "https://github.com/lattai">GitHub</a>
+                <a target ="_blank" class = "link" href= "https://www.linkedin.com/in/leah-attai-203">LinkedIn</a>
+                <a target ="_blank" class = "link" href= "https://github.com/lattai">GitHub</a>
```

Comment:
Really nice use of the target blank to ensure people don't loose sight of your website